macaroonbakery/macaroon.py

```diff
@@ -157,7 +167,38 @@ def _new_caveat_id(self, base):
         @param base string
         @return string
```

Comment:
I think this should be taking and returning bytes not strings.

Reply:
done.